src/icegrams/ngrams.py

```diff
@@ -864,7 +865,7 @@ def bigram_frequency(self, i0: Optional[int], i1: Optional[int]) -> int:
         if i0 is None or i1 is None:
             return 0
         # Check degenerate case
-        #if not (i0 or i1):
+        # if not (i0 or i1):
```

Comment:
Oops, this should not be commented - this is debugging stuff that was accidentally left in. Please de-comment the two lines here...